update_edge.py

```diff
@@ -0,0 +1,50 @@
+from graph import *
+
+def q2(g, source, destination, w):
+	src = g.V[source]
+	dest = g.V[destination]
+	src_visted_veritces=[src]
+	dest_visted_veritces=[dest]
+	while( dest not in src_visted_veritces and src not in dest_visted_veritces):
+		if dest.pi is not None:
+			 dest = dest.pi
+			 dest_visted_veritces.append(dest)
+		if src.pi is not None:
+			src = src.pi
+			src_visted_veritces.append(src)
+	
+	start_of_circle = dest  if dest in src_visted_veritces else src
+	src = g.V[source]
+	dest = g.V[destination]
+	max_edge = (Edge(source,destination), w[source][destination])
+	curr = Vertex(-1)
```

Comment:
It gets a value afterwards anyway

Reply:
so? not pre declare ?  py work that way?

Reply:
done